roles/ciao-image/meta/main.yml

```diff
@@ -0,0 +1,25 @@
+galaxy_info:
+  author: Erick Cardona
+  description: Ansible role to install the image service for a CIAO cluster
+  company: Intel
+
+  issue_tracker_url: https://github.com/01org/ciao
```

Comment:
We need to update the issue tracker url on all other roles too.
Issues for the ansible playbooks belong to github.com/clearlinux/clear-config-management